test/createTest.m

```diff
@@ -121,3 +121,28 @@ function createTest()
     fclose(FID);
 
 end
+
+function [result, index] = searchforType(type, graph) 
```

Comment:
Could we remove this function completely or is it needed somewhere in the tests?